### ISSUE TITLE
ORCiD is not a link

### ISSUE THREAD

**Issue by sven-mayer** [vtex-soft/texsupport.IOS-Book-Article#1]
On the first page, while the full link to the ORCiD is shown, one needs to copy it to access it. 

Proposed solution: use the `\href` command to make it clickable and save on space, as only the number can be shown, and the https://orcid.org/ link can be hidden. This is an advantage over `\url`, where the full URL needs to be visible for all authors on the first page of the paper.

**Comment by mingiss** [comment]
Hello.

Can't regenerate the issue. Have compiled the example with pdflatex of MikTeX 23.4 and TeX Live 2019 and 2021 -- all of them produce pdf with working links:
https://www.dropbox.com/scl/fi/vbgt9ze9k7xwz7qw8ih9r/IOS-Book-Article.zip

Regards, Mindaugas.

**Comment by sven-mayer** [comment]
I just asked for access to the Dropbox link. However, I compiled mine with Overleaf pdfLatex 2023 and it does not work with the current implementation – minimal reproducible example: https://www.overleaf.com/read/csvwqnzvfmmf#1969b5 – just hover over the link in the Overleaf preview. However, now with using the`\href` command in my pull request, this is how it looks like once you hover over the link:
<img width="377" alt="image" src="https://github.com/vtex-soft/texsupport.IOS-Book-Article/assets/12493208/6b6977ce-1127-4f43-ab88-b85796914a0a">

I know that most PDF readers (e.g., Adobe) autodetect URLs and make them clickable anyway. However, this can just be fixed by using the `\url`or `\href` command. Also of course, the proposed version is shorter as you do not need to have https://orcid.org/ for each author again.

**Comment by mingiss** [comment]
You are right, Adobe Reader recognizes the syntax of url links, my bad. I've added \href to the orcid output procedure, you could probably use it or correct according to your taste:
https://www.dropbox.com/scl/fi/8th8uuy4klqb4s92a5sdh/IOS-Book-Article.cls?rlkey=xgykuei68a555cob5lzx5ckb6&dl=0
(I hope this time the link is working.)
If the IOS authorities would ask, the correction could be added to the common template style as well. Just keep in mind, that it depends on the further workflow of the manuscripts -- for example, url references are cleaned up in print versions.
